resources/build/build-utils.sh

```diff
@@ -665,9 +735,9 @@ _builder_pad() {
 builder_display_usage() {
   local e program description
 
-  # Minimum padding is 12 characters, increase this if necessary
+  # Minimum padding is 13 characters, increase this if necessary
   # if you add other, longer, global options (like --verbose)
-  local width=12
+  local width=13
```

Comment:
These can be reset to 12

Reply:
Oh right, since we dropped the need for `--report-scope`.  That's fair.